apps/src/applab/designMode.js

```diff
@@ -473,7 +474,8 @@ designMode.parseFromLevelHtml = function(rootEl, allowDragging, prefix) {
   if (!Applab.levelHtml) {
     return;
   }
-  var levelDom = $.parseHTML(Applab.levelHtml);
+  var sanitized = sanitizeHtml(Applab.levelHtml);
```

Comment:
actually I think this is the case in which we had talked about opening a newrelic or honeybadger error. Here is a bug to track that: https://www.pivotaltracker.com/story/show/112187121